contrib/terraform/openstack/README.md

```diff
@@ -284,6 +284,7 @@ For your cluster, edit `inventory/$CLUSTER/cluster.tfvars`.
 |`master_server_group_policy` | Enable and use openstack nova servergroups for masters with set policy, default: "" (disabled) |
 |`node_server_group_policy` | Enable and use openstack nova servergroups for nodes with set policy, default: "" (disabled) |
 |`etcd_server_group_policy` | Enable and use openstack nova servergroups for etcd with set policy, default: "" (disabled) |
+|`additional_server_groups` | Extra server groups to create. Set "policy" to the policy for the group, default: [] (to not create any extra groups) |
```

Comment:
Could you specify the exact format here? Something like this (but with the actually correct format):
```suggestion
|`additional_server_groups` | Extra server groups to create. Set "policy" to the policy for the group, expected format is [{"new-server-group" = {"policy" = "hard-anti-affinity"}}], default: [] (to not create any extra groups) |
```